src/components/uploadPicture/uploadImage.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useState, useEffect, useCallback } from "react";
+import styles from "./uploadImage.module.css";
+import { trpc } from "@/client/lib/trpc";
+import axios from "axios";
+import { FaCamera, FaCheckCircle } from "react-icons/fa";
+
+interface UploadImageComponentProps {
+  title: string;
+  fileName: string;
+}
+
+const UploadImageComponent: React.FC<UploadImageComponentProps> = ({
+  title,
+  fileName,
+}) => {
+  const [showStart, setShowStart] = useState<boolean>(true);
+  const [showTakePicture, setShowTakePicture] = useState<boolean>(false);
+  const [showUploadPicture, setShowUploadPicture] = useState<boolean>(false);
+  const [startButtonDisabled, setStartButtonDisabled] =
+    useState<boolean>(false);
+  const [uploading, setUploading] = useState<boolean>(false);
+  const [uploadSuccess, setUploadSuccess] = useState<boolean>(false);
+
+  const [capturedImage, setCapturedImage] = useState<string>("");
+
+  const onClickStartButton = () => {
+    setShowStart(false);
+    setShowTakePicture(true);
+  };
+
+  const handleFile = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onload = () => {
+        if (typeof reader.result === "string") {
+          setCapturedImage(reader.result);
```

Comment:
I would recommend keeping the file as a Blob here and transforming later as necessary